README.md

```diff
@@ -1,47 +1,65 @@
-Tor. Please manually add the Tor Project's Apt repository. Instructions are
+**NOTICE:** Some distributions, including Ubuntu, do not maintain a secure and updated
+version of Tor. Please manually add the Tor Project's Apt repository. Instructions are
 here: https://www.torproject.org/docs/debian.html.en
 
 At the time of this writing, the Tor Project's signing key for Apt packages
 is: A3C4 F0F9 79CA A22C DBA8  F512 EE8C BC9E 886D DD89
 
 ## Parkbench Dependencies
-_torwatchdog_ depends on two other pieces of the Parkbench project, which must be installed first:
 
-1. [_confighelper_](https://github.com/park-bench/confighelper)
-2. [_gpgmailer_](https://github.com/park-bench/gpgmailer)
+_torwatchdog_ depends on two other Parkbench projects which must be installed first:
+
+1.  [_confighelper_](https://github.com/park-bench/confighelper)
+2.  [_gpgmailer_](https://github.com/park-bench/gpgmailer)
 
 ## Steps to Build and Install
 
-1.   Manually add the Tor Project's Apt repository to ensure you have a secure and updated version of Tor. Instructions located here: https://www.torproject.org/docs/debian.html.en
-2.   Clone the latest release tag. (Do not clone the master branch. `master` may not be stable.)
-3.   Use `debuild` in the project root directory to build the package.
-4.   Use `dpkg -i` to install the package.
-5.   Use `apt-get -f install` to resolve any missing dependencies. The daemon will attempt to start and fail. (This is expected.)
-6.   Locate the example configuration file at `/etc/torwatchdog/torwatchdog.conf.example`. Copy or rename this file to `torwatchdog.conf` in the same directory. Edit this file to enter the Tor URL you want to monitor or to modify other settings.
-7.   Restart the daemon with `service torwatchdog restart`. If the configuration file is valid and named correctly, the service will now start successfully.
+1.  Manually add the Tor Project's Apt repository to ensure you have a secure and updated
+    version of Tor. Instructions located here:
+    https://www.torproject.org/docs/debian.html.en
+2.  Clone the repository and checkout the lastest release tag. (Do not build against the
+    `master` branch. The `master` branch might not be stable.)
+3.  Use `debuild` in the project root directory to build the package.
+4.  Use `dpkg -i` to install the package.
+5.  Run `apt-get -f install` to resolve any missing dependencies. The daemon will attempt to
+    start and fail. (This is expected.)
+6.  Copy or rename the example configuration file
+    `/etc/torwatchdog/torwatchdog.conf.example` to `/etc/torwatchdog/torwatchdog.conf`. Edit
+    this file to enter the Tor URL you want to monitor. Other settings can also be modified.
+7.  Use `chmod` to clear the _other user_ permissions bits of `watchman.conf`. Namely, remove
+    read, write, and execute permissions for _other_.
+8.  To ease system maintenance, add `torwatchdog` as a supplimental group to administrative
```

Comment:
"Supplimental" is spelled supplemental.

Reply:
This was also a problem in the other projects.